include/seqan3/alphabet/detail/alphabet_proxy.hpp

```diff
@@ -233,26 +233,15 @@ class alphabet_proxy : public alphabet_base<derived_type,
     constexpr derived_type & assign_rank(alphabet_rank_t<alphabet_type> const r) noexcept
     {
         alphabet_type tmp{};
-        using seqan3::assign_rank_to;
-        assign_rank_to(r, tmp);
+        seqan3::assign_rank_to(r, tmp);
```

Comment:
I thought we agreed on not to use `seqan3::` namespace explicitly?

Reply:
The change was from before we discussed it.